Library/Formula/qjackctl.rb

```diff
@@ -0,0 +1,135 @@
+class Qjackctl < Formula
+  desc "simple Qt application to control the JACK sound server daemon"
+  homepage "http://qjackctl.sourceforge.net"
+  url "https://downloads.sourceforge.net/qjackctl/qjackctl-0.4.1.tar.gz"
+  sha256 "98024e1ea9f55ac16c290feda051d6494b2261e83f6a918f0fa583b509e2bfd1"
+  head "http://git.code.sf.net/p/qjackctl/code", :using=>:git
+
+  stable do
+    patch :DATA
```

Comment:
No its not possible to submit this patch, I'm sorry. All changesets in this patch are contained in upstream, as i already said before. Their commit hashes are contained in the respective part and can be used to verify this change in upstream.  
And there are also comments explaining their necessity (in the patch data part)

> Am 24.11.2015 um 11:25 schrieb Mike McQuaid <notifications@github.com>:
> 
> In Library/Formula/qjackctl.rb:
> 
> > @@ -0,0 +1,135 @@
> > +class Qjackctl < Formula
> > +  desc "simple Qt application to control the JACK sound server daemon"
> > +  homepage "http://qjackctl.sourceforge.net"
> > +  url "https://downloads.sourceforge.net/qjackctl/qjackctl-0.4.1.tar.gz"
> > +  sha256 "98024e1ea9f55ac16c290feda051d6494b2261e83f6a918f0fa583b509e2bfd1"
> > +  head "http://git.code.sf.net/p/qjackctl/code", :using=>:git
> > +
> > +  stable do
> > +    patch :DATA
> Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!
> 
> —
> Reply to this email directly or view it on GitHub.
>